[DO NOT MERGE]FRW-1441 Dynamic multi store integration guides (#1650)
spryker/spryker-docs: _includes/pbc/all/install-features/202304.0/install-the-dynamic-store-availability-notification-feature.md

> This document describes how to ingrate the Dynamic Store + Avalability Notification feature into a Spryker project.

Reviewer: ```suggestion
This document describes how to integrate the Dynamic Store + Availability Notification feature into a Spryker project.
```

## Install feature core

### Prerequisites

To start feature integration, integrate the required features:

| NAME | VERSION |
| --- | --- |
| Spryker Core | {{page.version}} |
| Availability Notification | {{page.version}} |

> ### 1) Set up configuration

Reviewer: ```suggestion
### Set up configuration
```

Add the following configuration to your project:

| CONFIGURATION  | SPECIFICATION | NAMESPACE |
| --- | --- | --- |
| AvailabilityNotificationConstants::REGION_TO_YVES_HOST_MAPPING (See below in `config/Shared/config_default.php`) | Defines regions to Yves host mapping. | Spryker\Shared\AvailabilityNotification |

**config/Shared/config_default.php**

```php

<?php

use Spryker\Shared\AvailabilityNotification\AvailabilityNotificationConstants;

$config[AvailabilityNotificationConstants::REGION_TO_YVES_HOST_MAPPING] = [
    'EU' => getenv('SPRYKER_YVES_HOST_EU'),
    'US' => getenv('SPRYKER_YVES_HOST_US'),
];

```

{% info_block warningBox "Verification" %}  

> To verify mail links are correct, make sure that the following configuration is correct:

Reviewer: ```suggestion
To verify the email links are correct, make sure that the following configuration is correct:
```
